dnf/automatic/main.py

```diff
@@ -91,8 +95,12 @@ def __init__(self, filename=None, downloadupdates=False,
 
         if downloadupdates:
             self.commands.download_updates = True
+        elif downloadupdates is False:
```

Comment:
False in self.commands.download_updates is default. Why such --no-installupdates option is good for.

Reply:
I am sorry I got it, it overrides setting in conf. Then ok